crates/ruff/src/rules/tryceratops/rules/raise_within_try.rs

```diff
@@ -83,9 +87,54 @@ pub(crate) fn raise_within_try(checker: &mut Checker, body: &[Stmt], handlers: &
         visitor.raises
     };
 
+    if raises.is_empty() {
+        return;
+    }
+
+    // The names of exceptions handled by the `try` Stmt. We can't compare `Expr`'s since they have
+    // different ranges, but by virtue of this function's call path we know that the `raise`
+    // statements will always be within the surrounding `try`.
```

Comment:
We should use `ComparableExpr` here -- it lets us compare expressions as value types (i.e., it ignores locations when doing the comparison).

Reply:
Oh, didn't know about that. Thanks!